core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -98,7 +98,7 @@ protected void inboundTransportError(Status errorStatus) {
     }
     // For transport errors we immediately report status to the application layer
     // and do not wait for additional payloads.
-    transportReportStatus(errorStatus, false, new Metadata.Trailers());
+    transportReportStatus(errorStatus, true, new Metadata.Trailers());
```

Comment:
According to the comment "For transport errors we immediately report status to the application layer and do not wait for additional payloads.", I thought this was a typo?